concepticondata/conceptlists/Goz-2017-600.tsv

```diff
@@ -0,0 +1,601 @@
+Goz-2017-600-413	413	pilot	pilot			2.66	1.95	2.41	3.1	3.1	6.08	6.35	51	2	5
+Goz-2017-600-414	414	pire	flea	232	FLEA	2.64	2.61	3.35	2.9	1.86	4.87	6.69	5	2	4
+Goz-2017-600-415	415	pirinç	rice	926	RICE	1.84	1.75	1.88	1.95	1.78	6.5	6.69	101	2	6
+Goz-2017-600-416	416	polis	police	656	POLICE	1.92	1.71	1.89	2.1	1.95	6.28	6.55	278	2	5
+Goz-2017-600-417	417	pomat	pomade			4.97	3.5	5.71	5.3	4.35	2.74	5.21	1	2	5
+Goz-2017-600-418	418	prensip	principle			4.91	3.28	5.4	5.83	5.04	2.85	1.98	61	3	7
+Goz-2017-600-419	419	protein	protein			4.01	2.97	3.73	4.42	4.8	3.29	5.17	64	4	7
+Goz-2017-600-420	420	punto	font size			5.8	3.5	5.7	6.24	6.09	3.32	4.92	1	2	5
+Goz-2017-600-421	421	pusu	ambush	65	AMBUSH	3.96	2.7	4.52	4.56	4.09	3.39	4.39	7	2	4
+Goz-2017-600-422	422	radyo	radio	1315	RADIO	2.11	1.79	2.09	2.04	2.48	6.47	6.75	140	2	5
+Goz-2017-600-423	423	radyum	radium			6.24	3.93	6.6	6.41	6.58	2.61	5.32	2	2	6
+Goz-2017-600-424	424	rahatsız	uneasy			2.33	1.85	2.66	2.54	2.33	2.9	2.6	21	3	8
+Goz-2017-600-425	425	rakam	digit			2.02	1.86	2.11	2.36	1.8	5.89	4.14	148	2	5
+Goz-2017-600-426	426	rakip	rival			2.91	2.21	3.01	3.57	2.87	4.04	4.7	117	2	5
+Goz-2017-600-427	427	reklâm	advertisement			2.97	1.68	2.18	3.5	4.28	5.06	5.39	211	2	6
+Goz-2017-600-428	428	rezalet	scandal			3.86	2.61	4.08	4.43	4.31	2.6	2.69	31	3	7
+Goz-2017-600-429	429	röfle	highlighting (hair)			5.47	3.3	5.67	6.36	6.02	3.81	5.94	1	2	5
+Goz-2017-600-430	430	rüzgâr	wind	960	WIND	1.83	1.7	1.86	2.03	1.75	5.14	6.09	185	2	6
+Goz-2017-600-431	431	sabır	patience			2.93	2.09	3.32	3.61	2.76	2.77	1.5	44	2	5
+Goz-2017-600-432	432	sadık	loyal			3.57	2.46	3.97	4.23	3.69	2.88	2.07	32	2	5
+Goz-2017-600-433	433	sadist	sadist			5.71	3.54	6.27	6.03	5.77	3.55	3.81	1	2	6
+Goz-2017-600-434	434	sağlam	robust			2.26	1.92	2.61	2.51	2.08	3.58	3.66	120	2	6
+Goz-2017-600-435	435	sahil	shore	113	SHORE	2.7	1.97	2.23	3.25	3.23	6.29	6.38	73	2	5
+Goz-2017-600-436	436	salata	salad			1.92	1.7	1.95	2.15	1.87	6.51	6.79	72	3	6
+Goz-2017-600-437	437	salon	hall			2.34	1.82	1.97	2.79	2.69	5.59	6.33	244	2	5
+Goz-2017-600-438	438	sanatçı	artist			2.63	2.03	2.38	3.27	2.79	4.5	5.14	219	3	7
+Goz-2017-600-439	439	sandık	chest (wooden)	3150	BOX OR CHEST	2.14	1.89	2.48	2.43	1.83	6.47	6.86	17	2	6
+Goz-2017-600-440	440	saray	palace			2.77	1.98	2.43	3.15	3.43	6.05	6.56	129	2	5
+Goz-2017-600-441	441	sarp	steep	2717	STEEP	4.76	3.46	5.8	5.21	4.2	3.82	4.44	1	1	4
+Goz-2017-600-442	442	sarsıcı	traumatic			4.16	2.73	4.11	4.91	4.76	2.68	2.68	4	3	7
+Goz-2017-600-443	443	sazlık	reed bed			3.93	2.67	4.75	4.43	3.76	4.93	6.52	2	2	6
+Goz-2017-600-444	444	sebze	vegetable	924	VEGETABLES	1.88	1.59	1.81	2.17	1.93	6.19	6.71	128	2	5
+Goz-2017-600-445	445	seçmen	voter			4.28	2.92	4.92	4.71	4.54	4.82	6.2	52	2	6
+Goz-2017-600-446	446	selâm	greeting			2.05	1.68	2.26	2.32	1.98	4.21	4.13	67	2	5
+Goz-2017-600-447	447	semer	saddle	3065	SADDLE	3.43	2.88	5.14	3.42	2.46	4.38	6.42	1	2	5
+Goz-2017-600-448	448	serbest	free			2.76	2.03	2.69	3.34	2.94	2.37	2.09	108	2	7
```

Comment:
2316 FREE?

Reply:
There are two words translated with "free" in the list. This one refers to the "gratis" meaning of free.